locust/stats.py

```diff
@@ -439,18 +539,21 @@ def median_from_dict(total, count):
 """
 
 def on_request_success(request_type, name, response_time, response_length):
-    global_stats.get(name, request_type).log(response_time, response_length)
+    #global_stats.get(name, request_type).log(response_time, response_length)
```

Comment:
should this be deleted instead of commented?

Reply:
Ah, yes :)!